src/components/SuccessRateContextProvider.tsx

```diff
@@ -77,17 +100,16 @@ const parseProbeResults = (probes: Probe[] | undefined, data: any) => {
     return acc;
   }, {});
 
+  // TODO: Re-type latency check data
```

Comment:
Is this still TODO? Fine if so and you want to do a follow up PR

Reply:
Still TODO, hoping for future self to come across it 😛